app/src/test/java/com/onelist/CategoryUnitTest.kt

```diff
@@ -7,7 +7,7 @@ import org.junit.Test
 class CategoryUnitTest {
 
     @Test
-    fun categoryToString() {
+    fun `toString returns "1 - Category Name" when categoryID = 1 & name = "Category Name"`() {
```

Comment:
Kotlin allows spaces in function names which should only be used for tests. I have updated all of the unit test names to follow this convention "method name - expected behavior - when condition" which makes the test more readable and specific. If more unit tests are written to test different cases or functions this format can be used. 

Link to source: https://betterprogramming.pub/android-unit-testing-choosing-naming-convention-and-test-cases-d1a3122ac28a

Reply:
Great!